src/thimble_server/data_access_layer/sql/select/select_post.sql

```diff
@@ -0,0 +1,2 @@
+-- selects post information for a given post ID
+SELECT file, poster FROM posts WHERE id = :postid;
```

Comment:
i'd rather return too much information than not enough, just also return that column. it doesn't hurt. it opens up more ways to use the function down the road.